### ISSUE TITLE
Cache is deleted regardless of --file-cache-timeout-in-seconds

### ISSUE THREAD

**Issue by dashesy** [Azure/azure-storage-fuse#409]
### Which version of the blobfuse was used?
1.2.1

### Which OS (please include version) are you using?

Ubuntu 16

### What problem was encountered?

I am using `--file-cache-timeout-in-seconds=345600` to get a cache that lasts longer, but the cache is removed regardless.

### Have you found a mitigation/solution?

No

### By default, blobfuse logs errors to syslog. If this is relevant, is there anything in the syslog that might be helpful?

No errors

### If relevant, please share your mount command.

```bash
blobfuse ~/blobs/storage --container-name=detectortrain --tmp-path=/tmp/blobcache/storage --file-cache-timeout-in-seconds=345600
```

**Comment by vibhansa-msft** [comment]
In the latest code I see file not being refreshed until the given expiry time is hit. You can re-validate your observation in next release or with latest code.

**Comment by NaraVen** [comment]
Ehsan,   When the available memory is < 80% items get evicted from cache. Can you check how much space you have available df -l when you see cache getting deleted. Also, repeating what Vikas has commented above, the latest code seems to not have any issues. Even after an hour I see items retrieved from cache.

**Comment by dashesy** [comment]
Yes disk was close to 80% full, but 20% of few terrabytes is still plenty of room. Can this behavior be set using a flag? and then empty cache using LRU.  

Thanks. I will try the latest version, as soon as it is available in `apt-get`

**Comment by NaraVen** [comment]
Setting the cache_size in bytes as a config value is a feature currently in development. It will be part of the next release,  in a month or so. Even with the latest code there is no option currently to set cache size.

**Comment by vibhansa-msft** [comment]
To control size of cache '--cache-size-mb' command line option is available from version 1.3.2.
